euphoria-hbase/src/main/java/cz/seznam/euphoria/hbase/HFileSink.java

```diff
@@ -206,38 +225,46 @@ public HFileSink build() {
 
   }
 
-  private final String table;
+  private final String tableName;
   private final boolean doBulkLoad;
   private final byte[][] endKeys;
   @Nullable
   private final Windowing<Cell, ?> windowing;
   private final UnaryFunction<Window<?>, String> folderNaming;
   private final HadoopSink<ImmutableBytesWritable, Cell> rawSink;
 
+  private final String hdfsUser;
+  private final String hbaseUser;
```

Comment:
This seems unused. Why do we do `chmod a+rwx` instead of changing ownership of the resulting hfiles?

Reply:
changed... for now the copied files are the correct user (hbase) and the temporary directorie have RWX so the calling user can delete them... this was problem of previous implementation,that there was permission error after chown